src/components/AdjMatrix/AdjMatrixMethods.ts

```diff
@@ -324,11 +314,7 @@ export class View {
         this.hideToolTip();
         this.unHoverEdge(d);
       })
-      .on('click', (d: Cell, i: number, nodes: any) => {
-        const interaction = select(nodes[i]).attr('class');
-        const action = this.changeInteractionWrapper(interaction, d);
-        this.provenance.applyAction(action);
-      })
+      .on('click', (d: Cell) => this.selectElement(d))
```

Comment:
I believe you can simplify this to:

```suggestion
      .on('click', this.selectElement)
```